# Message containing the sensor information of the sats car
std_msgs/Header header

VehicleState front
VehicleState rear
VehicleState left
VehicleState right

#GpsData vec
